core/src/nodes/tasks/manager.rs

```diff
@@ -237,7 +237,12 @@ impl<I, O, H, E, HE, T, C> Manager<I, O, H, E, HE, T, C> {
             let msg = ToTaskMessage::HandlerEvent(event.clone());
             match task.sender.start_send(msg) {
                 Ok(()) => {},
-                Err(ref err) if err.is_full() => {},        // TODO: somehow report to user?
+                Err(ref err) if err.is_full() => {
+                    // Note that the user is expected to call `poll_ready_broadcast` beforehand,
+                    // which returns `Poll::Ready` only if the channel isn't full. Reaching this
+                    // path always indicates a mistake in the code.
+                    log::warn!("start_broadcast called while channel was full");
```

Comment:
Why not:

```suggestion
                    log::warn!("start_broadcast called while channel was full. Have you called `poll_ready_broadcast` before?");
```

Reply:
That's very internal function that newcomers really aren't expected to use, but why not.